### PR TITLE
Heap and ABI functions moved aesophia -> aebytcode

### PR DESCRIPTION
**Pull request by hanssv**
Need latest aesophia in #2314 - so this does the vital parts pointing aeternity to aeb_xxx instead och aeso_xxx...

### CODE REVIEW

**Comment by rvirding** [comment]
This should finish [PT-164597852](https://www.pivotaltracker.com/story/show/164597852)

File: rebar.config [hunk 1]
```diff
@@ -60,10 +60,10 @@
                    {ref, "9ff46af"}}},
 
         {aebytecode, {git, "https://github.com/aeternity/aebytecode.git",
-                     {ref,"2d599d"}}},
+                     {ref,"9041423"}}},
```
Comment (ThomasArts) [inline rebar.config]: That's not latest master, but I will bump soon as well. You now did the majority of the work already

Comment (hanssv) [reply rebar.config]: It is exactly the version that the referenced `aesophia` points to, that is correct I think?!

**Comment by hanssv** [comment]
> This should finish [PT-164597852](https://www.pivotaltracker.com/story/show/164597852)

I'm not familiar with the scope of the ticket, but unless it doesn't also deal with making `aesophia` a test-only dependency this might be it?!

**Pull request merged**